integrations/dynatrace/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,72 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: problem
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .problemId
+          title: .title
+          blueprint: '"dynatraceProblem"'
+          properties:
+            entityTags: .entityTags[].stringRepresentation
+            evidenceDetails: .evidenceDetails.details[].displayName
+            managementZones: .managementZones[].name
+            problemFilters: .problemFilters[].name
+            severityLevel: .severityLevel
+            status: .status
+            startTime: ".startTime / 1000 | todate"
+            endTime: ".endTime | if . == -1 then null else (./1000 | todate) end"
+          relations:
+            impactedEntities: .impactedEntities[].entityId.id
+            linkedProblemInfo: .linkedProblemInfo.problemId
+            rootCauseEntity: .rootCauseEntity.name
+
+  - kind: entity
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .entityId
+          title: .displayName
+          blueprint: '"dynatraceEntity"'
+          properties:
+            firstSeen: ".firstSeenTms / 1000 | todate"
+            lastSeen: ".lastSeenTms / 1000 | todate"
+            type: .type
+            tags: .tags[].stringRepresentation
+            managementZones: .managementZones[].name
+            properties: .properties
+            fromRelationships: .fromRelationships
+            toRelationships: .toRelationships
+  - kind: slo
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"dynatraceSlo"'
+          properties:
+            status: .status
+            target: .target
+            enabled: .enabled
+            burnRateMetricKey: .burnRateMetricKey
+            warning: .warning
+            error: .error
+            errorBudget: .errorBudget
+            errorBudgetBurnRate: .errorBudgetBurnRate.burnRateValue
+            errorBudgetMetricKey: .errorBudgetMetricKey
+            evaluatedPercentage: .evaluatedPercentage
+            evaluationType: .evaluationType
+            filter: .filter
+            metricExpression: .metricExpression
+            metricKey: .metricKey
+            normalizedErrorBudgetMetricKey: .normalizedErrorBudgetMetricKey
+            relatedOpenProblems: .relatedOpenProblems
+            relatedTotalProblems: .relatedTotalProblems
+            timeframe: .timeframe
```

Comment:
you are exporting almost the whole rest api response - https://docs.dynatrace.com/docs/dynatrace-api/environment-api/service-level-objectives/get-slo#response-body-json-model

I think this needs to be reduced. not a blocker in any way, remember that this is only the template, the users will be able to extend it and if they will want they will add more properties.

lets try to reduce the number of properties that we return to only the most relevant ones. If you are not sure which one, we can discuss it.